src/field_theory/minpoly/basic.lean

```diff
@@ -54,6 +54,17 @@ lemma ne_zero [nontrivial A] (hx : is_integral A x) : minpoly A x ≠ 0 :=
 lemma eq_zero (hx : ¬ is_integral A x) : minpoly A x = 0 :=
 dif_neg hx
 
+lemma minpoly_alg_hom (f : B →ₐ[A] B') (hf : function.injective f) (x : B) :
+  minpoly A (f x) = minpoly A x :=
+begin
+  rw [minpoly, minpoly],
```

Comment:
Maybe there's a way to prove this without unfolding, but this was pretty easy anyway.